test/integration/http_integration.h

```diff
@@ -27,6 +27,8 @@ class IntegrationCodecClient : public Http::CodecClientProd {
                          Http::CodecClient::Type type);
 
   IntegrationStreamDecoderPtr makeHeaderOnlyRequest(const Http::RequestHeaderMap& headers);
```

Comment:
I think you can just add an optional parameter to the original function but default it to false.

Reply:
good to know, I was unaware of this :-) I set the default to `true` to keep the behaviour unchanged